package.json

```diff
@@ -345,6 +327,15 @@
         ".nycrc"
       ],
       "output": []
+    },
+    "link-check": {
```

Comment:
it probably doesn't hurt, but it won't run often since humans don't do much with the top-level cli